packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -30,6 +21,7 @@ export interface GraphNGProps extends Themeable {
   width: number;
   height: number;
   data: DataFrame[];
+  alignedData: DataFrame;
```

Comment:
I keep these two, for now, I guess the `alignedData` should be sufficient, but who knows what the future holds. Tooltip/ContextMenu use the aligned frame, as they can rely on the series index from uPlot, but I have encountered some weirdness when using with PlotLegend.